### DISCUSSION TITLE
after reboot from fresh install - Spiderfoot Container continuous restarting

### DISCUSSION THREAD

**Discussion by Accla89** [telekom-security/tpotce#1012] [General]
Hello together,

I have installed a tpot via tpot image. (Create your own ISO Image)
The Installation Types: Standard
After the installation, all services are available.

VM Spec:

12GB RAM (10,3GB Used)
4vCPUs
32GB SSD

Then after a reboot of the VM, Spiderfoot does not start anymore.
Container is Continuous restarting

Greeting

**Comment by t3chn0m4g3** [comment]
Please try to log the Spiderfoot events `docker logs spiderfoot -f`.
If that does not help stop the T-Pot services `systemctl stop tpot`, go to `/opt/tpot/docker/spiderfoot` and run it individually `docker-compose up`.
Post the logs here.

**Comment by Accla89** [comment]
Traceback (most recent call last):
  File "/home/spiderfoot/spiderfoot/db.py", line 306, in __init__

*************************************************************
 Use SpiderFoot by starting your web browser of choice and 
 browse to http://127.0.0.1:8080
*************************************************************

    self.dbh.execute('SELECT COUNT(*) FROM tbl_scan_config')
sqlite3.DatabaseError: database disk image is malformed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/spiderfoot/spiderfoot/db.py", line 347, in create
    self.dbh.execute(qry)
sqlite3.OperationalError: table tbl_event_types already exists

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/spiderfoot/spiderfoot/db.py", line 311, in __init__
    self.create()
  File "/home/spiderfoot/spiderfoot/db.py", line 361, in create
    raise IOError(f"SQL error encountered when setting up database: {e.args[0]}")
OSError: SQL error encountered when setting up database: table tbl_event_types already exists

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/spiderfoot/sf.py", line 603, in <module>
    main()
  File "/home/spiderfoot/sf.py", line 236, in main
    start_web_server(sfWebUiConfig, sfConfig)
  File "/home/spiderfoot/sf.py", line 575, in start_web_server
    cherrypy.quickstart(SpiderFootWebUi(sfWebUiConfig, sfConfig), script_name=web_root, config=conf)
  File "/home/spiderfoot/sfwebui.py", line 75, in __init__
    dbh = SpiderFootDb(self.defaultConfig)
  File "/home/spiderfoot/spiderfoot/db.py", line 314, in __init__
    raise IOError(f"Tried to set up the SpiderFoot database schema, but failed: {e.args[0]}")
OSError: Tried to set up the SpiderFoot database schema, but failed: SQL error encountered when setting up database: table tbl_event_types already exists


**stop the T-Pot Service does not help**

/opt/tpot/docker/spiderfoot]$ docker-compose up
Creating network "spiderfoot_spiderfoot_local" with the default driver
Creating spiderfoot ... done
Attaching to spiderfoot
spiderfoot    | 2022-02-14 10:28:48,719 [INFO] Starting web server at 0.0.0.0:8080 ...
spiderfoot    | 2022-02-14 10:28:48,738 [WARNING] 
spiderfoot    | ********************************************************************
spiderfoot    | Warning: passwd file contains no passwords. Authentication disabled.
spiderfoot    | Please consider adding authentication to protect this instance!
spiderfoot    | Refer to https://www.spiderfoot.net/documentation/#security.
spiderfoot    | ********************************************************************
spiderfoot    | 
spiderfoot    | Traceback (most recent call last):
spiderfoot    |   File "/home/spiderfoot/spiderfoot/db.py", line 306, in __init__
spiderfoot    | 
spiderfoot    | *************************************************************
spiderfoot    |  Use SpiderFoot by starting your web browser of choice and 
spiderfoot    |  browse to http://127.0.0.1:8080
spiderfoot    | *************************************************************
spiderfoot    | 
spiderfoot    |     self.dbh.execute('SELECT COUNT(*) FROM tbl_scan_config')
spiderfoot    | sqlite3.DatabaseError: database disk image is malformed
spiderfoot    | 
spiderfoot    | During handling of the above exception, another exception occurred:
spiderfoot    | 
spiderfoot    | Traceback (most recent call last):
spiderfoot    |   File "/home/spiderfoot/spiderfoot/db.py", line 347, in create
spiderfoot    |     self.dbh.execute(qry)
spiderfoot    | sqlite3.OperationalError: table tbl_event_types already exists
spiderfoot    | 
spiderfoot    | During handling of the above exception, another exception occurred:
spiderfoot    | 
spiderfoot    | Traceback (most recent call last):
spiderfoot    |   File "/home/spiderfoot/spiderfoot/db.py", line 311, in __init__
spiderfoot    |     self.create()
spiderfoot    |   File "/home/spiderfoot/spiderfoot/db.py", line 361, in create
spiderfoot    |     raise IOError(f"SQL error encountered when setting up database: {e.args[0]}")
spiderfoot    | OSError: SQL error encountered when setting up database: table tbl_event_types already exists
spiderfoot    | 
spiderfoot    | During handling of the above exception, another exception occurred:
spiderfoot    | 
spiderfoot    | Traceback (most recent call last):
spiderfoot    |   File "/home/spiderfoot/sf.py", line 603, in <module>
spiderfoot    |     main()
spiderfoot    |   File "/home/spiderfoot/sf.py", line 236, in main
spiderfoot    |     start_web_server(sfWebUiConfig, sfConfig)
spiderfoot    |   File "/home/spiderfoot/sf.py", line 575, in start_web_server
spiderfoot    |     cherrypy.quickstart(SpiderFootWebUi(sfWebUiConfig, sfConfig), script_name=web_root, config=conf)
spiderfoot    |   File "/home/spiderfoot/sfwebui.py", line 75, in __init__
spiderfoot    |     dbh = SpiderFootDb(self.defaultConfig)
spiderfoot    |   File "/home/spiderfoot/spiderfoot/db.py", line 314, in __init__
spiderfoot    |     raise IOError(f"Tried to set up the SpiderFoot database schema, but failed: {e.args[0]}")
spiderfoot    | OSError: Tried to set up the SpiderFoot database schema, but failed: SQL error encountered when setting up database: table tbl_event_types already exists
spiderfoot exited with code 1

**Reply by t3chn0m4g3** [reply]
Run ` echo > /data/spiderfoot/spiderfoot.db` and `reboot`.

**Reply by Accla89** [reply]
Spiderfoot running, other Service also running.

thanks @t3chn0m4g3

**Reply by t3chn0m4g3** [reply]
Fixed with https://github.com/telekom-security/tpotce/commit/518c56def97ea9c941a22e0d6138dde2626f6f06
